qt/scientific_interfaces/ISISReflectometry/GUI/Batch/RowProcessingAlgorithm.cpp

```diff
@@ -251,7 +253,13 @@ std::unique_ptr<MantidQt::API::IAlgorithmRuntimeProps> createAlgorithmRuntimePro
   updateExperimentProperties(*properties, model.experiment());
   updateInstrumentProperties(*properties, model.instrument());
   // Update properties from the wildcard row in the lookup table
-  updateLookupRowProperties(*properties, model.findWildcardLookupRow());
+  if (auto lookupRow = model.findWildcardLookupRow()) {
+    g_log.warning("Using experiment settings from the wildcard row.");
+    updateLookupRowProperties(*properties, *lookupRow);
+  } else {
+    g_log.warning("No experiment settings found; using algorithm defaults. You may wish to specify a wildcard"
```

Comment:
Do we want these warnings to replace the ones in LookupTable.h? At the moment they are somewhat duplicated when running a regular, non-live reduction

Reply:
I've updated the test instructions with some tests for a normal reduction as well as the live data tests.